### ISSUE TITLE
COA Tools on Blender 3 / Windows 10

### ISSUE THREAD

**Issue by gameyakuza** [ndee85/coa_tools#191]
I tried all packages and the fix from neirenoir, neither work.

**Comment by BluePrintRandom** [comment]
try 3.1x there was a fix by @hoogly Boogly for custom bone shape drawing

**Comment by BluePrintRandom** [comment]
https://builder.blender.org/download/daily/

**Comment by gameyakuza** [comment]
It's a great plugin, but it's unusable as long as it doesn't work from the project's page itself. I can't use multiple version of Blender just for a plugin. Neirenoir mentioned something that was never fixed in the base plugin, what is it?

**Comment by ndee85** [comment]
Hi @gameyakuza 
what does not work? Installing isn't working, how do you install the addon? Just tested with a freshly installed Blender 3.0 and the 2.8 branch is working fine.
Please make sure to follow the installation instruction.

**Comment by gameyakuza** [comment]
Yes, it worked. Don't know what I did differently, I had the trace callback error happening yesterday, like in 2.9.

**Comment by ktorenz** [comment]
> Hi @gameyakuza what does not work? Installing isn't working, how do you install the addon? Just tested with a freshly installed Blender 3.0 and the 2.8 branch is working fine. Please make sure to follow the installation instruction.

I tried to install the addon in Blender 3.0, but unfortunately the addon doesn't appear in Tools section.

**Comment by ktorenz** [comment]
Now I can't import a sprite.

![image](https://user-images.githubusercontent.com/98167700/150545582-d746100d-5bf9-40b3-9b17-6d7cc9941cc8.png)

**Comment by Tenpi** [comment]
In 3.2 it seems to install without any errors, but I can't find the addon anywhere in the toolbar. I hope that you update this for new Blender versions I don't really want to downgrade to 2.8 and lose great 3.0 features.

**Comment by 3dcinetv** [comment]
I also get multiple errors. I did use it up to Blender 2.92 and it worked fine. 
Is ther any chance to continue to use it for Blender 3.5 and beyond?

**Comment by SuitableForWork** [comment]
For me it only works up to 2.93.14 and when I export to dragonbones or creature it crashes instantly.

But this branch worked for me: https://github.com/kreezii/coa_tools

I can't find anything that works on 3.0 and above.

I would love to see this continued on anyway, since additive and multiply modes seem to be missing.

I wish someone would pick this up :(
